### PR TITLE
Add support for 13.6.4

### PR DESCRIPTION
**Pull request by r3dlobst3r**
This PR will add support for users on Ventura 13.6.4.

### CODE REVIEW

**Comment by jetfir3** [comment]
The 13.6.4 offsets seem to match all of the previous 13.6.x releases, only the binary hash changes.

PR should only need the following:
```
	// macOS 13.6.4
        hexToByte32("8f22dcfda56a4d3c38931f20fe33db1a6720e4d8571e452aa5a8b56b4c69842a"): offsets_13_6,
```

**Comment by bderagon** [comment]
looks right to me, can we get this merged?

File: nac/offsets.go [hunk 1]
```diff
@@ -238,6 +238,8 @@ var offsets = map[[32]byte]imdOffsetTuple{
 	hexToByte32("fff8db27fef2a2b874f7bc6fb303a98e3e3b8aceb8dd4c5bfa2bad7b76ea438a"): offsets_13_6,
 	// macOS 13.6.3
 	hexToByte32("2c674438d30bf489695f2d1b8520afc30cbfb183af82d2fc53d74ce39a25b24e"): offsets_13_6,
+	// macOS 13.6.4
+        hexToByte32("8f22dcfda56a4d3c38931f20fe33db1a6720e4d8571e452aa5a8b56b4c69842a"): offsets_13_6_4,
```
Comment (jetfir3) [inline nac/offsets.go:242]: This should be `offsets_13_6,` and not `offsets_13_6_4,` (due to the recent removal of superfluous offsets)

**Review by xChickens — APPROVED** [review]

**Comment by xChickens** [comment]
Your commit has spaces and not tabs so the lint is failing

File: nac/offsets.go [hunk 2]
```diff
@@ -238,6 +238,8 @@ var offsets = map[[32]byte]imdOffsetTuple{
 	hexToByte32("fff8db27fef2a2b874f7bc6fb303a98e3e3b8aceb8dd4c5bfa2bad7b76ea438a"): offsets_13_6,
 	// macOS 13.6.3
 	hexToByte32("2c674438d30bf489695f2d1b8520afc30cbfb183af82d2fc53d74ce39a25b24e"): offsets_13_6,
+	// macOS 13.6.4
+        hexToByte32("8f22dcfda56a4d3c38931f20fe33db1a6720e4d8571e452aa5a8b56b4c69842a"): offsets_13_6,
```
Comment (xChickens) [inline nac/offsets.go:242]: Spaced using spaces instead of tabs

**Review by xChickens — CHANGES_REQUESTED** [review]

**Comment by 0xdevalias** [comment]
Duplicate PR?

- https://github.com/beeper/mac-registration-provider/pull/35

**Pull request closed without merging**